webapp/components/user_settings/manage_languages.jsx

```diff
@@ -36,9 +36,12 @@ export default class ManageLanguage extends React.Component {
         });
     }
     submitUser(user) {
+        this.setState({isSaving: true});
```

Comment:
Can you add an initial value for `this.state.isSaving` to the state object created in the constructor (line 23)?